src/usage.h

```diff
@@ -13,6 +13,7 @@ R"(
     skafos fetch --table <table_name>
     skafos kill [<project_token>] [--deployments <deployment_ids>] [--job_ids <job_ids>]
     skafos remote info [<project_token>]
+    skafos orgs [<name>] [--default]
     skafos -h | --help
     skafos -v | --version
 Commands:
```

Comment:
I would add a command description for users to get a quick overview of what the command does, if they were to call the skafos help command.

Reply:
I made this change in #41 in b9e5fba